TCAT/Views/SummaryView.swift

```diff
@@ -27,13 +27,11 @@ class SummaryView: UIView {
         // View Initialization
         super.init(frame: .zero)
 
-        /*
-         TODO:
-         This value ends up getting overwritten by constraints, which is what we want,
-         but for some reason if it is not set prior to writing the constraints, the
-         entire view comes out blank. I'm still investigating but it seems to be an,
-         issue with the Pulley Pod that we're using.
-        */
+        // TODO:
+        // This value ends up getting overwritten by constraints, which is what we want,
+        // but for some reason if it is not set prior to writing the constraints, the
+        // entire view comes out blank. I'm still investigating but it seems to be an,
+        // issue with the Pulley Pod that we're using.
```

Comment:
What is the reason that we don't want to use /**/ for multiline comments?

Reply:
i'm following Google's style guidelines and `/** */` is used for Java docs so we shouldn't use them for Swift